ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.13-periodics.yaml

```diff
@@ -323,87 +323,6 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
-- agent: kubernetes
```

Comment:
could you please specify a quick comment about why you're cleaning this job in the PR?

Reply:
For this change, it is automatically generated by `Make job`